docs/tutorials/stateless-application/expose-external-ip-address.md

```diff
@@ -0,0 +1,151 @@
+---
+---
+
+{% capture overview %}
+
+This page shows how to create a Kubernetes Service object that exposees an
+external IP address.
+
+{% endcapture %}
+
+
+{% capture prerequisites %}
+
+* Install [kubectl](http://kubernetes.io/docs/user-guide/prereqs).
+
+* Use a cloud provider like Google Container Engine or Amazon Web Services to
+  create a Kubernetes cluster.
```

Comment:
I think we should mention why: That we are using type Load Balancer. Also, maybe link to http://kubernetes.io/docs/user-guide/load-balancer/ Hopefully that page can be improved to cover the details of the GCE/AWS implementations.